roles/patch_host_config/templates/patch-worker-partition.j2

```diff
@@ -0,0 +1,71 @@
+{
+	"ignition": {
+		"version": "3.2.0"
```

Comment:
We are going to need to come up with a way to bypass this functionality for older version of OCP, as ignition spec 3.2 is not supported on 4.6 (and possibly 4.7). I suggest adding something to the inventory validation step, and documenting the `partitions` field in the sample inventory.